packages/css-color-parser/test/wpt/color-invalid-relative-color.mjs

```diff
@@ -24,12 +24,6 @@ const tests = [
 
 	// hsl(from ...)
 
-	// Testing invalid permutation (types don't match).
-	['hsl(from rebeccapurple s h l)'],
-	['hsl(from rebeccapurple s s s / s)'],
-	['hsl(from rgb(10%, 20%, 30%, 40%) s h l)'],
-	['hsl(from rgb(10%, 20%, 30%, 40%) s s s / s)'],
-
 	// Testing invalid values.
```

Comment:
These tests change for invalid, unparseable color notations.
With the specification change, the color is no longer invalid.